### PR TITLE
Address feedback on gecko.rst

### PR DESCRIPTION
**Pull request by zbraniecki**
@Pike, @stasm: r?

### CODE REVIEW

File: docs/gecko.rst [hunk 1]
```diff
@@ -38,19 +38,19 @@ Create An FTL Translation Resource
 Translations for L20n are stored in ``.ftl`` files.  You can find more 
 information about the FTL format at the end of this document.
```
Comment (Pike) [inline docs/gecko.rst]: This still refers to the inline ftl section we removed.

Move the link up to here?

File: docs/gecko.rst [hunk 2]
```diff
@@ -38,19 +38,19 @@ Create An FTL Translation Resource
...
 
-The FTL files leverage the existing packaging infrastructure for locales.  The 
-*en-US* translations for ``browser`` can be found in 
-``browser/locale/en-US/browser``, as specified by ``browser/locale/jar.mn``::
+The FTL files leverage the existing packaging infrastructure for locales.  The
+translations for ``browser`` can be found in
+``browser/locale/*/browser``, as specified by ``browser/locale/jar.mn``::
```
Comment (Pike) [inline docs/gecko.rst]: Jein ;-)

Packaging FTL files in Firefox is done by adding their directory to jar.mn. Please only package complete directory structures, and don't try to move files around.

The top-level directory like `browser` is close to where your code is. Inside `browser/locales/en-US`, the first directory clarifies the target audience of the feature.

File: docs/gecko.rst [hunk 3]
```diff
@@ -38,19 +38,19 @@ Create An FTL Translation Resource
...
 
     browser                                        (%browser/*.ftl)
```
Comment (Pike) [inline docs/gecko.rst]: [localization] @AB_CD@.jar:
        browser    (%browser/**/*.ftl)

is what we use now.

File: docs/gecko.rst (hunk 2 above)
Comment (stasm) [inline docs/gecko.rst]: I don't understand the `don't try to move files around` part.  Can you elaborate?

Comment (stasm) [inline docs/gecko.rst]: Maybe rephrase like so:

    Please only package complete directory structures, and keep 
    the same directory layout as in the source.
